docs/sources/community/maintaining/release/_index.md

```diff
@@ -39,5 +39,6 @@ to publish a new [Grafana Loki](https://github.com/grafana/loki) release.
 1. [Backport commits]({{< relref "./backport-commits" >}})
 1. [Prepare release notes]({{< relref "./prepare-release-notes" >}})
 1. [Prepare changelog]({{< relref "./prepare-changelog" >}})
+1. [Update version numbers]({{< relref "./update-version-numbers" >}})
```

Comment:
Does the referenced page exist already?

Reply:
yeah michel. major and minor release guides already refer to this page